Looks good, but @bruvzg had another error on macOS, could you fix it too?
```
scene/3d/voxel_light_baker.cpp:1032:15: error: call to 'pow' is ambiguous
                float att = Math::pow(1.0 - angle / p_spot_angle, p_spot_attenuation);
                            ^~~~~~~~~
```
(should also be `1.0f`)

--- a/scene/3d/baked_lightmap.cpp
+++ b/scene/3d/baked_lightmap.cpp
@@ -254,7 +254,7 @@ bool BakedLightmap::_bake_time(void *ud, float p_secs, float p_progress) {
 	if (time - btd->last_step > 1000000) {
 
 		int mins_left = p_secs / 60;
-		int secs_left = Math::fmod(p_secs, 60.0);
+		int secs_left = Math::fmod(p_secs, 60.0f);
 		int percent = p_progress * 100;
 		bool abort = bake_step_function(btd->pass + percent, btd->text + " " + itos(percent) + "% (Time Left: " + itos(mins_left) + ":" + itos(secs_left) + "s)");
 		btd->last_step = time;

--- a/scene/3d/voxel_light_baker.cpp
+++ b/scene/3d/voxel_light_baker.cpp
@@ -1029,7 +1029,7 @@ void VoxelLightBaker::plot_light_spot(const Vector3 &p_pos, const Vector3 &p_axi
 			continue; // too far away
 		}
 
-		float att = Math::pow(1.0 - angle / p_spot_angle, p_spot_attenuation);
+		float att = Math::pow(1.0f - angle / p_spot_angle, p_spot_attenuation);
 
 		{
 			float d = light_pos.distance_to(to);
@@ -1638,9 +1638,9 @@ Vector3 VoxelLightBaker::_compute_ray_trace_at_pos(const Vector3 &p_pos, const V
 
 	for (int i = 0; i < samples; i++) {
 
-		float random_angle1 = (((rand() % 65535) / 65535.0) * 2.0 - 1.0) * spread;
+		float random_angle1 = (((Math::rand() % 65535) / 65535.0) * 2.0 - 1.0) * spread;
 		Vector3 axis(0, sin(random_angle1), cos(random_angle1));
-		float random_angle2 = ((rand() % 65535) / 65535.0) * Math_PI * 2.0;
+		float random_angle2 = ((Math::rand() % 65535) / 65535.0) * Math_PI * 2.0;
 		Basis rot(Vector3(0, 0, 1), random_angle2);
 		axis = rot.xform(axis);
 

